appveyor.yml

```diff
@@ -6,16 +6,13 @@ install:
   # Get the latest stable version of Node.js or io.js
   - ps: Install-Product node $env:nodejs_version
   - ps: $env:BuildNumber= $env:APPVEYOR_BUILD_NUMBER
+  - ps: $env:Configuration="Release"
 init:
   - git config --global core.autocrlf true
 build_script:
-  - dotnet restore
```

Comment:
The Visual Studio 2017 image has everything already to build.
the current script works, tests are also passing.
It might be better but I would like to understand then.